huntsman/utils/__init__.py

```diff
@@ -1,17 +1,41 @@
 import os
 from pocs.utils import listify
 from pocs.utils.config import load_config as config_loader
+from panoptes.utils.config import client
+from huntsman.utils.pyro import get_own_ip
 
+#==============================================================================
 
+def query_config_server(logger=None, *args, **kwargs):
```

Comment:
Actually this entire function is redundant with: https://github.com/panoptes/panoptes-utils/blob/develop/panoptes/utils/config/client.py#L6
which already catches errors.

I think you just need `config = client.get_config(ip_address)` (note I removed kwargs as `get_config` doesn't accept it.